cookbooks/bcpc-hadoop/templates/default/hv_hive-env.sh.erb

```diff
@@ -6,3 +6,4 @@ export HIVE_CONF_DIR=/etc/hive/conf.<%= node.chef_environment %>
 # export HIVE_AUX_JARS_PATH=
 export HIVE_AUX_JARS_PATH=/usr/lib/hcatalog/share/hcatalog/hcatalog-core.jar
 
+export HADOOP_HEAPSIZE=<%=node[:bcpc][:hadoop][:heapsize] %>
```

Comment:
I am not sure if we really want to do that. Specifying it in `hdp_hadoop-env.sh.erb` will impact other hadoop daemons as well. With this PR we still set it to default causing no impact to any other process. Since `hive-env.sh` will include this as environment overridden value, it will only impact hive processes.